src/lib/issue.ts

```diff
@@ -69,14 +69,14 @@ export class Issue {
     this.state = issue.state;
     this.comment = issue.comment;
     this.discussion = issue.discussion;
-    this.labels = issue.labels;
+    this.tags = issue.tags;
     this.timestamp = issue.timestamp;
   }
 
   // Counts the amount of comments and replies in a discussion
   countComments(): number {
     return this.discussion.reduce((acc, comment) => {
-      if (comment.replies) return acc + comment.replies.length + 1; // We add all replies and 1 forathe comment in this loop.
+      if (comment.replyTo) return acc + comment.replyTo.length + 1; // We add all replies and 1 forathe comment in this loop.
```

Comment:
IIRC, `replyTo` is not an array anymore in the new model, but an index. Maybe keep it as `.replies` ?

Reply:
4067a2c03c830c7f7263c70bc2a434df083134ab